index.d.ts

```diff
@@ -44,8 +51,8 @@ range.next().value;
 range.next().value;
 //=> 2
 
-console.log(...getRange({start: 0, end: -5, step: -1}));
-//=> [0, -1, -2, -3, -4]
+console.log(...getRange({start: 0, end: -5, step: -1, inclusive: true}));
+//=> [0, -1, -2, -3, -4, -5]
```

Comment:
I would prefer to not change this, but instead add a similar example (line 54 + 55) in the `readonly inclusive?: boolean;` docs. Readme too.